main/domoticz.cpp

```diff
@@ -854,7 +854,7 @@ int main(int argc, char**argv)
 				return 1;
 			}
 			std::string wwwport = cmdLine.GetSafeArgument("-sslwww", 0, "");
-			auto iPort = std::stoi(wwwport);
+			int iPort = std::stoi(wwwport);
```

Comment:
@keryell , @gizmocuz , I am ok with using `int` here, but (as a not C++ programmer) it got me thinking about the use/usefulness of `auto`.

I can see why in a case like this using `auto` makes sense. A new variable gets initialized and is directly populated by a _conversion_ function. This conversion function is _casting_ in a sense from one type to another. The _conversion_ function (`std::stoi`) has an intended destination type so it makes sense to initialize the new variable with the same type. That is what `auto` is intended for and the compiler is very well aware if the output type of this _conversion_ function. We humans are maybe not, at least not directly. That is why coding it explicit as `int` makes it clear to us.

**But** it can also be read as another _implicit_ conversion if the _conversion_ function would be of a different type than `int`.

It can be argued that, although in the (_C_) past everything should be very explicit, in modern C++ it is fine to be _less explicit_ when it comes to certain declaration, etc. as the compiler knows how to cope with this and coding can be done at a higher abstraction similar to languages like Python or Go.

Just my 2 cents..